arezoo/script.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+let apples = 20;
+let oranges = 30;
+let mangoes = 5;
+console.log(apples == oranges);
+console.log(apples != oranges);
+console.log(oranges > apples);
+console.log(mangoes * apples > mangoes + oranges);
```

Comment:
console.log((mangoes * apples) > (mangoes + oranges));

same with the rest

Reply:
Yes it is.  That's why Visual Studio itself deletes extra code(()) that is not needed.